pkgs/tools/misc/rpm-ostree/default.nix

```diff
@@ -28,6 +28,8 @@ stdenv.mkDerivation rec {
     libcap ostree rpm glib systemd polkit bubblewrap
     json-glib libarchive libsolv librepo
     pcre check python
+     # libdnf
+    json_c libmodulemd_1 utillinux sqlite cppunit
```

Comment:
Why is this required? I do not see `libmodulemd_1` to be used anywhere.

Reply:
The comment above says `libdnf` because there's the subproject https://github.com/rpm-software-management/libdnf/tree/8a6b5837bb868ee398cd5bfc76a1d9668da0a050 that I believe depends on it.

Reply:
You are right, it is in the submodule which I did not have checked out.